american-chemical-society.csl

```diff
@@ -255,8 +255,8 @@
             <text variable="URL"/>
             <date variable="accessed" prefix="(accessed " suffix=")" delimiter=" ">
```

Comment:
The delimiter attribute here sets those spaces.
What do the guidelines say these dates should look like? (I don't have access to the guidelines)

Reply:
Thanks for fixing this!

According to the ACS manual (https://pubs.acs.org/doi/full/10.1021/acsguide.40305), the recommended format for general websites is:

> Author or Organization (if any). _Title of Site_. URL (accessed YYYY-MM-DD).

Actually, on a related note, based on that new format, the title of site should be updated to include the suffix as "." and the title of the site should be in italics.